src/main/java/taskAT7/LoggerManager.java

```diff
@@ -0,0 +1,32 @@
+package taskAT7;
+
+import org.apache.logging.log4j.Level;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.apache.logging.log4j.core.config.Configurator;
+
+import java.util.Scanner;
+
+public class LoggerManager {
+    private final Logger logger = LogManager.getLogger(LoggerManager.class);
+
+    public LoggerManager() {
+        setLogLevel();
+    }
+
+    public void log(String message) {
+        logger.log(this.logger.getLevel(), message);
+    }
+
+    public void info(String message) {
+        logger.info(message);
+    }
+
+    public void setLogLevel() {
+        Scanner sc = new Scanner(System.in);
+        System.out.println("Enter log level");
+        String logLevel = sc.next();
```

Comment:
it would be better if console message would be more descriptive - because I have no idea about format that I need stick to, in order to have correct input